### DISCUSSION TITLE
Issue with uploading MonAI label through 3D slicer to orthanc server

### DISCUSSION THREAD

**Discussion by nikakula98** [Project-MONAI/MONAILabel#1792] [Q&A]
Hello,

I am using a virtual machine through my school to run 3D slicer and MonAI label. I am able to pull up CT scan images from an Orthanc server where my images are saved. However, upon creating a segmentation and attempting to submit the label to the Orthanc server, I receive an internal server error. I have a screenshot of my work attached. Thanks in advance for any help!
![image](https://github.com/user-attachments/assets/9d616a1b-0658-402b-8d3d-3741a6bfc02e)

**Comment by lassoan** [comment]
Probably developers will need the server log.

**Comment by diazandr3s** [comment]
Thanks, @lassoan 

@nikakula98, please share the MONAI Label logs to understand what could be the issue
